src/util/articleRenderers.ts

```diff
@@ -111,7 +111,7 @@ const renderMCSArticle = (json: any, dump: Dump, articleId: string, articleDetai
         json.remaining.sections
             .forEach((oneSection: any, i: number) => {
                 // if below is to test if we need to nest a subsections into a section
-                if (oneSection.toclevel === 1) {
+                if (oneSection.toclevel === 1 || oneSection.toclevel === 0) {
```

Comment:
* Why we keep `toclevel === 1`?
* What are the boundaries of tolevel values?
* What is this test doing here exactly, the comment above does not make it clear to me, can you please rewrite it?

Reply:
Other Mediawikis that I have come across have their TOC sections starting from level 1 and sub-sections in these wiktionary start with level 2. For eg. en.wiktionary.

https://en.wiktionary.org/api/rest_v1/page/mobile-sections/Latin

On the other hand, ru.wiktionary has TOC section starting from level 0 and its sub-sections start from 1.

https://ru.wiktionary.org/api/rest_v1/page/mobile-sections/%D0%92%D0%B8%D0%BA%D0%B8%D1%81%D0%BB%D0%BE%D0%B2%D0%B0%D1%80%D1%8C

I checked both kind of Mediawikis and adding the check wouldn't affect existing wikis and would help us handle sections for wikis having TOC sections starting with 0.

Reply:
@bakshiutkarsha Why some are starting with toclvel == 1 and other tolevel == 2? maybe the wiki code is not the same?